api/mailer/mailer.js

```diff
@@ -53,4 +53,16 @@ router.post('/validateVerificationEmail', async (req, res) => {
     });
 });
 
+router.post('/getCalendarEvents', async (req, res) => {
```

Comment:
should we make this a get request? after all, we're *getting* the events hahaha funny joke original content
```suggestion
router.get('/getCalendarEvents', async (req, res) => {
```